Infrastructure/network.tf

```
@@ -3,7 +3,7 @@ locals {
     dev     = "Dev"
     test    = "Test"
     prod    = "Prod"
-    sandbox = "Sandbox"
+    stag = "Tools"
```

```suggestion
    stage = "Tools"
```
everywhere if there is no impact of this change